frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -406,16 +410,19 @@ function ReaderTypeset:onSetPageMargins(margins, silent)
     end
     self.ui.document:setPageMargins(left, top, right, bottom)
     self.ui:handleEvent(Event:new("UpdatePos"))
-    if not silent then
+    if refresh_callback then
         -- Show a toast on set, with the unscaled & scaled values
         UIManager:show(InfoMessage:new{
             text = T(_([[
     Margins set to:
```

Comment:
@poire-z 
Is this intentionally all indented? It seems like it should be:

```lua
Margins set to

    indented
    indented

Click to dismiss
```

Reply:
That was from @Niluje PR #4945 (to which I added "Tap to dismiss").
(I like it all indented, and I think you once lectured me that if I wanted this visual effect, I'd have to use some left padding - and I went just removing the indentation :)
So, no opinion :)

Reply:
I don't *think* I made that decision consciously, FWIW, so, err, no opinion either ;).

Reply:
In that case I'll get rid of it. It just adds additional complexity to translating the string.